### PR TITLE
improve about dialog

### PR DESCRIPTION
**Pull request by Simon-Laux**
- add more versions to about dialog and update order of existing information.

Bildschirmfoto 2024-02-11 um 18 28 32

### CODE REVIEW

**Comment by farooqkz** [comment]
You must understand when you make CI unhappy, you make me unhappy. And when you make me unhappy, I nag all the day for you to death.

File: src/renderer/components/dialogs/About.tsx [hunk 1]
```diff
@@ -41,7 +41,11 @@ export function DCInfo(_props: any) {
...
   return (
     <>
-      <h3>Version details:</h3>
+      <h3>Debug Info for System &amp; Selected Account</h3>
+      <p>
+        Local Account Id:{' '}
+        <b>{window.__selectedAccountId || '(no account selected)'}</b>
+      </p>
       <div className='dialog-about__dc-details'>
         <table>
```
Comment (farooqkz) [inline src/renderer/components/dialogs/About.tsx:50]: Usually it's a good idea to use a CSS grid or flexbox rather than a table.

Comment (Simon-Laux) [reply src/renderer/components/dialogs/About.tsx:50]: it's a debug view we shouldn't spend to much time on it right now.

Comment (Simon-Laux) [reply src/renderer/components/dialogs/About.tsx:50]: also in this case the info fits well into a table IMO

**Comment by Simon-Laux** [comment]
CI should be fixed now.

> But do you think it makes sense to merge it after my Button refactor PR?

it merges into the sidebar pr

File: src/renderer/components/dialogs/About.tsx (hunk 1 above)
Comment (adzialocha) [reply src/renderer/components/dialogs/About.tsx:50]: Yeah! Tables were meant for ... tables! So it's actually a perfect use case!

File: src/renderer/components/dialogs/About.tsx [hunk 2]
```diff
@@ -67,6 +71,18 @@ export function DCInfo(_props: any) {
 export default function About({ onClose }: DialogProps) {
   const tx = useTranslationFunction()
 
+  const [coreVersion, setCoreVersion] = useState('')
+  const [sqliteVersion, setSqliteVersion] = useState('')
+
+  useEffect(() => {
+    BackendRemote.rpc.getSystemInfo().then(info => {
+      console.warn({ info })
```
Comment (adzialocha) [inline src/renderer/components/dialogs/About.tsx:79]: I'd use `info` as a log level here. If you want to hide it from the developer by default, use `debug`, that's still not a warning but optional

**Review by adzialocha — APPROVED** [review]
Minor change request for the log level, otherwise all good and helpful. Thank you!

File: src/renderer/components/dialogs/About.tsx (hunk 2 above)
Comment (Simon-Laux) [reply src/renderer/components/dialogs/About.tsx:79]: ah good catch it should be removed, I just used it for debugging

**Pull request merged**